src/res/translation/translation_nl_NL.ts

```diff
@@ -4476,62 +4476,62 @@ We hebben uw kanaal gedempt en &apos;Demp mijzelf&apos; geactiveerd. Los eerst h
     <message>
         <location filename="../../util.cpp" line="1893"/>
         <source>Released under the GNU General Public License version 2 or later (GPLv2)</source>
-        <translation type="unfinished"></translation>
+        <translation>Gereleased onder de GNU General Public License version 2 or later (GPLv2)</translation>
```

Comment:
s/version 2 or later/versie 2 of recenter/

Reply:
Corrected in commit **9f39e4d**